Azure/sonic-utilities
# [Mellanox] Add SDK dump with new SAI implementation

#### What I did
Add SDK dump output into techsupport in the generate_dump script.

#### How I did it
Add a step to copy SDK dump output into techsupport in the generate_dump script.

#### How to verify it
Simulate SDK event and check that dump is copied into the techsupport  results.

This PR is only valid after this PR is merged - https://github.com/DavidZagury/sonic-buildimage/pull/8
#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
